### ISSUE TITLE
Create 0.4.0 release?

### ISSUE THREAD

**Issue by urso** [elastic/go-licenser#34]
go-licenser v0.3.0 can't be installed currently, due to some dependency changing its path. The master branch seems to be usable.

```
go get: gopkg.in/src-d/go-license-detector.v2@v2.0.1 updating to

	gopkg.in/src-d/go-license-detector.v2@v2.0.4 requires

	gopkg.in/russross/blackfriday.v2@v2.0.1: parsing go.mod:

	module declares its path as: github.com/russross/blackfriday/v2

	        but was required as: gopkg.in/russross/blackfriday.v2
```

**Comment by marclop** [comment]
I released version `v0.3.1`. It should work well

**Comment by kvch** [comment]
It is indeed working for us. Thank you!
